packages/gasket-plugin-metadata/lib/utils.js

```diff
@@ -60,7 +55,6 @@ function safeAssign(target, other) {
 function loadAppModules(loader, app, modules) {
   app.modules = app.modules || [];
   Object.keys(app.package.dependencies)
-    .filter(name => !isPlugin(name) && !isPreset(name))
```

Comment:
Until we fully re-work this plugin, we should continue to filter out plugins and presets in this step. You could do so with a basic regexp like this:
```js
const rePluginPresets = /@gasket.+(plugin|preset)/;

// -- snipped --
  Object.keys(app.package.dependencies)
    .filter(name => !rePluginPresets.test(name))
```